src/Magento/FunctionalTestingFramework/Util/Env/EnvProcessor.php

```diff
@@ -0,0 +1,99 @@
+<?php
```

Comment:
We have a similar type of class as a 3rd party library, why did we write a new one?

Reply:
Discussed, future implementaion will get rid of this dependency.